srcpkgs/gmsh/template

```diff
@@ -0,0 +1,44 @@
+# Template file for 'gmsh'
+pkgname=gmsh
+version=4.0.7
+revision=1
+wrksrc=${pkgname}-${version}-source
+build_style=cmake
+configure_args="
+ -DENABLE_BUILD_SHARED=1
+ $(vopt_if numpy '-DENABLE_NUMPY=1' '-DENABLE_NUMPY=0')
+ $(vopt_if zipper '-DENABLE_ZIPPER=1' '-DENABLE_ZIPPER=0')"
+hostmakedepends="gcc-fortran"
+depends="python"
+short_desc="Three-dimensional finite element mesh generator"
+maintainer="Nathan Owens <ndowens04@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://gmsh.info"
+distfiles="https://gmsh.info/src/gmsh-${version}-source.tgz"
+checksum=c6572320d0ffdf7d2488e113861bc4bd9c38a29f7fc5b67957f6fbcb63fbdbd5
+
+build_options="numpy zipper"
+desc_option_numpy="Enable fullMatrix and numpy array conversion"
+desc_option_zipper="Enable zip file compression/decompression"
+
+# Remove builddir from destdir
+post_install() {
+	rm -rf buildir
+}
+
+libgmsh_package() {
+	short_desc+=" - library"
+	pkg_install() {
+		vmove "usr/lib/*.so.*"
+	}
```

Comment:
wrong ident level for `}`

Reply:
Looking at the template even in emacs, } looks to be aligned with pkg_install() ; which I've seen in other templates?